### PR TITLE
[VnV Plan] Add NFR tests for Usability and Humanity Requirements

### LINKED ISSUE #187: [VnV Plan] Nonfunctional Requirements System Tests - Usability and Humanity Requirements
**Report:** Verification and Validation Plan
**Section Number:** 5.2

### Rubric Sections
- Tests for Nonfunctional Requirements are specific
- Nondynamic testing  used as necessary
- Quality System Tests for Funct and NonFunc Requirements
- LO_Plan: Selects appropriately from relevant knowledge base to plan appropriate data collection methods and analysis strategies

### PR DESCRIPTION
**Pull request by LeonSo7**
Add NFR tests for Usability and Humanity Requirements

Closes #187

### CODE REVIEW

File: docs/VnVPlan/VnVPlan.tex [hunk 1]
```diff
@@ -392,9 +393,44 @@ \subsubsection{Module ?}
 
 \end{enumerate}
 
-\subsubsection{Module ?}
+\subsubsection{Usability and Humanity Requirements}
+\begin{enumerate}
 
-...
+	\item{NFRT-UH-1\\}
```
Comment (arkinmodi) [inline docs/VnVPlan/VnVPlan.tex:399]: I was doing it this way so the IDs are bolded and `...UH1` to reference that is it for the NFR UH1.
```suggestion
	\item \textbf{NFRT-UH1}
```

File: docs/VnVPlan/VnVPlan.tex [hunk 2]
```diff
@@ -392,9 +393,44 @@ \subsubsection{Module ?}
...
+
+	How test will be performed: The tester disconnect from the internet while the application is open.
+
+	\item{NFRT-UH-3\\}
+
+	Type: Dynamic, Manual
+
+	How test will be performed: The testers will complete the manual system tests for functional
+	requirements listed section \ref{Tests for Functional Requirements} on a MacOS desktop/laptop
+	device, a Windows desktop/laptop, an iOS mobile device, and an android mobile device.
```
Comment (arkinmodi) [inline docs/VnVPlan/VnVPlan.tex:431]: Which requirement is this referencing? I don't think this if for UH3. I think we should follow the format of `NFRT-UH3` to make traceability clearer (and my work later of creating the traceability matrix easier).

File: docs/VnVPlan/VnVPlan.tex [hunk 3]
```diff
@@ -392,9 +393,44 @@ \subsubsection{Module ?}
...
+	Type: Dynamic, Manual
+
+	Initial State: Device is connected to the internet and the application is open
+
+	Input/Condition: User disconnects from the internet
+
+	Output/Result: The system notifies the user that there is no network connection and functionality
+	will be limited.
+
+	How test will be performed: The tester disconnect from the internet while the application is open.
```
Comment (arkinmodi) [inline docs/VnVPlan/VnVPlan.tex:423]: This one looks like it is referencing multiple requirements. Can you leave a comment for which ones? (Mainly for me when I create the traceability matrix. For label I would go with the first requirement if fulfills (i.e., NFRT-UH2).

File: docs/VnVPlan/VnVPlan.tex (hunk 2 above)
Comment (LeonSo7) [reply docs/VnVPlan/VnVPlan.tex:431]: For some requirements, I'm guessing we might want more than one test; or one test might cover more than one requirement. I've added a comment for which requirement each non-functional test corresponds to.

File: docs/VnVPlan/VnVPlan.tex (hunk 3 above)
Comment (LeonSo7) [reply docs/VnVPlan/VnVPlan.tex:423]: Added a comment for each non-functional requirement test indication which requirement it maps to

**Review by arkinmodi — APPROVED** [review]

**Pull request merged**